test/include/materials/DerivativeMaterialInterfaceTestClient.h

```diff
@@ -30,9 +30,9 @@ class DerivativeMaterialInterfaceTestClient : public DerivativeMaterialInterface
 public:
   DerivativeMaterialInterfaceTestClient(const InputParameters & parameters);
 
-protected:
-  void computeProperties();
+  virtual void computeProperties();
 
+protected:
```

Comment:
Interesting that you fixed these access levels.. I was thinking that we should make these methods protected in the base class. They are highly unlikely to work if called publically. This is fine for now.